docs/Installation.md

```diff
@@ -784,8 +766,8 @@ Remove the database file and recreate it following the installation instructions
 [PostgreSQL instructions on Debian]: #b2-postgresql-installation-on-debian-and-ubuntu
 [PostgreSQL instructions on FreeBSD]: #b3-postgresql-installation-on-freebsd
 [Post-installation]: #6-post-installation
-[README.md-upgrade]: /README.md#upgrade
 [Upgrade database]: #7-upgrade-zonemaster-database
+[upgrade document]: docs/Upgrade.md

```

Comment:
It resides in the same directory so it should be (initial upper case is best):
```
[Upgrade document]: docs/Upgrade.md
```

Reply:
sould be fixed